torchtext/vocab.py

```diff
@@ -30,10 +30,7 @@ class Vocab(object):
         itos: A list of token strings indexed by their numerical identifiers.
     """
 
-    # TODO (@mttk): Populate classs with default values of special symbols
-    UNK = '<unk>'
-
-    def __init__(self, counter, max_size=None, min_freq=1, specials=['<unk>', '<pad>'],
+    def __init__(self, counter, max_size=None, min_freq=1, unk_token='<unk>', specials=['<pad>'],
```

Comment:
Should `unk_token` argument be placed at the end? Not sure if this will introduce a BC here.

Reply:
The order of kwargs is irrelevant so no (they are essentially treated as a dict).

Reply:
Not very end but behind `specials`?

Reply:
I can move it, sure.